### PR TITLE
Interface: Set favicon to ship sigil

### PR DESCRIPTION
**Pull request by tylershuster**
You didn't know you wanted it

### CODE REVIEW

**Comment by matildepark** [comment]
My main reservation is that statusbar is a really weird place to put document-wide logic. A "setfavicon" function in App.js and calling it onMount might be better. I'll leave it to @tacryt-socryp to review this though.

**Comment by tylershuster** [comment]
You're right, that *is* a really weird place to put it

**Comment by tylershuster** [comment]
This crash looks like an error

**Comment by matildepark** [comment]
Yeah, it just "took too long" — JS work very rarely trips CI.

File: pkg/interface/src/App.js [hunk 1]
```diff
@@ -79,6 +81,21 @@ class App extends React.Component {
     this.api.local.setDark(e.matches);
   }
 
+  setFavicon() {
+    if (window.ship.length < 14) {
+      const svg = sigiljs({
+        patp: window.ship,
+        renderer: stringRenderer,
+        size: 16,
+        colors: ['#fff', '#000']
```
Comment (matildepark) [inline pkg/interface/src/App.js:90]: Consider: we know our root identity's colours from our `window.ship` in `props.contacts['/~/default']`. Might be worth polling that?

**Comment by tylershuster** [comment]
How do y'all feel about making a broader change to use [React Helmet](https://www.npmjs.com/package/react-helmet) for:
- Favicon
- Title
- - Dynamically updating to reflect chat name
- Manifest
- ???

CC @urcades

**Comment by tylershuster** [comment]
Ok I did it anyway. But now I'm getting some weird merge conflicts that I'm not familiar with. Did I mess it up?

**Comment by matildepark** [comment]
I'm not seeing any merge conflicts here.

**Comment by matildepark** [comment]
Scope creep on this is a little large ... I'd just undo the Helmet commit.

**Comment by tylershuster** [comment]
Ok, I'm not sure what happened with the rebase, but the final version looks kosher now

**Pull request merged**